core/java/com/google/instrumentation/stats/RpcConstants.java

```diff
@@ -218,6 +221,128 @@
           DistributionAggregationDescriptor.create(RPC_BYTES_BUCKET_BOUNDARIES),
           Arrays.asList(RPC_SERVER_METHOD));
 
+  // Interval Stats
+  static final Duration MINUTE = Duration.create(60, 0);
+  static final Duration HOUR = Duration.create(60 * 60, 0);
+
+  /**
+   * Census defined rpc client {@link IntervalViewDescriptor}s.
+   */
+  public static final IntervalViewDescriptor RPC_CLIENT_ROUNDTRIP_LATENCY_INTERVAL_VIEW =
+      IntervalViewDescriptor.create(
+          "rpc client roundtrip latency",
+          "Latency in msecs",
+          RPC_CLIENT_ROUNDTRIP_LATENCY,
+          IntervalAggregationDescriptor.create(Arrays.asList(MINUTE, HOUR)),
+          Arrays.asList(RPC_CLIENT_METHOD));
+
+  public static final IntervalViewDescriptor RPC_CLIENT_REQUEST_BYTES_INTERVAL_VIEW =
+      IntervalViewDescriptor.create(
+          "rpc client request_bytes",
+          "Request MB",
+          RPC_CLIENT_REQUEST_BYTES,
+          IntervalAggregationDescriptor.create(Arrays.asList(MINUTE, HOUR)),
+          Arrays.asList(RPC_CLIENT_METHOD));
+
+  public static final IntervalViewDescriptor RPC_CLIENT_RESPONSE_BYTES_INTERVAL_VIEW =
+      IntervalViewDescriptor.create(
+          "rpc client response_bytes",
+          "Response MB",
+          RPC_CLIENT_RESPONSE_BYTES,
+          IntervalAggregationDescriptor.create(Arrays.asList(MINUTE, HOUR)),
+          Arrays.asList(RPC_CLIENT_METHOD));
+
+  public static final IntervalViewDescriptor RPC_CLIENT_ERROR_COUNT_INTERVAL_VIEW =
+      IntervalViewDescriptor.create(
+          "rpc client error count",
+          "RPC Errors",
+          RPC_CLIENT_ERROR_COUNT,
+          IntervalAggregationDescriptor.create(Arrays.asList(MINUTE, HOUR)),
+          Arrays.asList(RPC_CLIENT_METHOD));
```

Comment:
Interval stats do not include it internally. If it's necessary, we can add it later.